devel/addon/kyverno/policy.yaml

```diff
@@ -178,7 +178,7 @@ spec:
       resources:
         kinds:
         - Pod
-    name: priviledged-containers
+    name: privileged-containers
```

Comment:
I hope this is ok

Reply:
I think this change is fine since it's just a human-readable name that copied over from Kyverno's [disallow-privileged-containers.yaml](https://github.com/kyverno/policies/blob/b6aa5dbdca71cf7ac4e0cd09d97b9fc72e067212/pod-security/baseline/disallow-privileged-containers/disallow-privileged-containers.yaml#L15).